src/docs/development/ios-14.md

```diff
@@ -46,6 +48,67 @@ This crash _doesn't_ happen when using the iOS simulator.
 For more information,
 see [Issue 62531]({{site.github}}/flutter/flutter/issues/62531).
 
+## Launching Flutter with Flutter tools
+
+Due to low-level changes in iOS’s debugger mechanisms,
+developers using versions of Flutter earlier than 1.22 beta
+won't be able to launch apps (by using `flutter run`
+or a Flutter IDE plugin) on physical iOS devices
+running iOS 14. This affects **debug**, **profile**, and
+**release** builds. Simulator builds, add-to-app modules,
+and running directly from Xcode are unaffected. 
+
+Developers who have upgraded to Flutter 1.22 beta can build,
+test, and deploy to iOS without issue.
+
+## Debugging Flutter
+
+Due to added security around local network permissions in
+iOS 14, a permission dialog box must now be accepted for
+each application in order to enable Flutter debugging
+functionalities such as hot-reload and DevTools. 
+
+![Screenshot of "allow network connections" dialog]({% asset development/device-connect.png @path %})
+
+This affects debug and profile builds only and won't
+appear in release builds. 
+
+For add-to-app users, one additional step has been added
+to the add-to-app project setup guide,
+[Integrate a Flutter module into your iOS project][],
+to re-enable flutter attach for debug builds on physical
+devices on iOS 14. 
+
+[Integrate a Flutter module into your iOS projecta]: /docs/development/add-to-app/ios/project-setup
+
+## Launching debug Flutter without a host computer
+
+Also due to changes in debugger mechanisms, once a Flutter
+debug application is installed on the device
+(either by using `flutter run` a Flutter enabled IDE,
+or from Xcode), the application can no
+longer be re-launched by tapping the application’s icon
+in the home screen in iOS 14 on physical devices. 
+
+Other launch paths without a host computer, such as deep links
+or notifications, won't work on iOS 14 physical devices in debug mode. 
+
+Add-to-app debug mode modules will crash on iOS 14
+physical devices when running the [`FlutterEngine`][].
```

Comment:
Jenn made a good point. Append ` if the host application is launched from the home screen`.